source/_posts/2024-01-17-voice-assistant-contest.markdown

```diff
@@ -0,0 +1,109 @@
+
+In fact, we believe that what we built during the Year of the Voice allows almost anyone to build a voice assistant that embodies our Open Home values perfectly: Personalized, Private and Sustainable.
+Every component of our voice assistants can be heavily customized to fit your language, your style and your needs, fully local options exist for each of them, and finally you can retrofit voice assistant into anything, even a [1970s Walkie Talkie](https://www.youtube.com/shorts/HqYbhk6vq9E) found in a flea market
+
+This is why, after spending a year building great things, we want to celebrate what our voice assistants can enable with the community: We’re launching a voice assistant contest, with great prizes to win!
+
+## Contest Specifics
+
+The contest will be held between Today (the **17th of January**) and the **10th of March**.
+
+We tried to be as inclusive as possible. The contest is not only about building voice assistant hardware, it is also about building voice experiences that can be used with any voice assistant (For example, [a blueprint that you can use to generate images from voice and cast them on your TV](https://my.home-assistant.io/redirect/blueprint_import?blueprint_url=https%3A%2F%2Fwww.home-assistant.io%2Fblueprints%2Fblog%2F2023-08%2Fcast_dall_e.yaml): “Show me a picture of an astronaut riding a unicorn!”)
+
+An entry in the contest can take any form: A video, a blog post, a website, an event, etc.
+
+The only requirement is for you to post on [this forum category](https://community.home-assistant.io/c/voice-assistant-contest/60), this is how you officially participate in the contest.
+
+Winners will be announced on the **15th of March**.
+
+A closing ceremony will take place on the **10th of April** in the form of a livestream. Winners will be invited!
+
+<p><img src='/images/blog/2024-01-11-voice-assistant-contest/contest_timeline.png' class='no-shadow' /></p>
+
+## Contest Categories
+
+You can participate in the contest in four different categories.
+
+### Most creative satellite ideas
+
+We have all the ingredients to unleash creativity: Custom sentences, custom wake words, custom voices. This category is here to showcase your smartest ideas.
+We will be judging the creativity, uniqueness, and coherency of the whole package.
+
+Example: [Piitaya’s](https://github.com/piitaya) moving and talking droid.
+
+<lite-youtube videoid="vQ7Hmeume9g" videotitle="Wake word demonstration on ESPHome-based 3D printed droid in Home Assistant | Year of the Voice"></lite-youtube>
+
+### Best Starting Guides
+
+Because our voice assistant can be heavily customized, starting can be daunting. This category is here to showcase the easiest ways to create a voice assistant for a novice. From ordering parts, flashing the firmware, integrating it into Home Assistant, and using it.
```

Comment:
```suggestion
Because our voice assistant can be heavily customized, starting can be daunting. This category showcases the easiest ways to create a voice assistant for a novice. From ordering parts, flashing the firmware, integrating it into Home Assistant, and using it.
```